README.md

```diff
@@ -63,7 +63,7 @@ Follow the instructions in the [Prerequisites](#magic_wand-prerequisites) to con
 | [Getting Started](./scenarios/getting_started/) | This quick start scenario simulates basic MQTT tasks.| &check;| &check;| &check;|
 | [Telemetry](./scenarios/telemetry/)  | This scenario simulates multiple clients (the producers) sending data to a different set of topics to be consumed by a single application (the consumer). | &check;| &check;| &check;|
 | [Command](./scenarios/command/)  | This scenario simulates the request-response messaging pattern using MQTT v5.  | &check;| &check; | soon|
-| [Alert](./scenarios/alert/)  | This scenario simulates a fan-out use case where multiple clients receive a singlemessage from the same topic.  | soon| soon| soon|
+| [Alert](./scenarios/alert/)  | This scenario simulates a fan-out use case where multiple clients receive a singlemessage from the same topic.  |  &check;| soon| soon|
```

Comment:
Minor suggestion. Maybe instead of "simulates", use "illustrate".

Reply:
I'd prefer to keep _simulate_ as the real scenario might have additional concerns, such as security.